addons/OPT/GPS/fn_clientInit.sqf

```diff
@@ -16,105 +17,30 @@ GVAR(processedIcons) = [];
 GVAR(lastProcessedIcons) = [];
 
 
+// Create markers for all players already in here.
+{
+    if ([_x] call FUNC(isUnitVisible)) then {
+        private _iconId = toLower format [QGVAR(IconId_Player_%1_%2), _x, group _x isEqualTo group CLib_Player];
+        DUMP("UNIT ICON ADDED: " + _iconId);
```

Comment:
Würde das in eine Funktion `getUnitIconID` oder so packen. Das ist ja was, was du öfter brauchst und wenn du da szentral an einer Stelle hast, dann kann das später auch recht einfach geändert werden (vorausgesetzt du benutzt die Funktion dann auch an allen Stellen :) )